inference-engine/src/mkldnn_plugin/nodes/mkldnn_extract_image_patches_node.h

```diff
@@ -62,15 +66,60 @@ class MKLDNNExtractImagePatchesNode : public MKLDNNNode {
     std::vector<size_t> _ksizes;
     std::vector<size_t> _strides;
     std::vector<size_t> _rates;
-    size_t _pad_left;
-    size_t _pad_top;
-    std::shared_ptr<jit_uni_extract_image_patches_kernel> extract_image_patches_kernel;
     static const std::set<size_t> _supported_precisions_sizes;
-
     ExtImgPatcherPadType _auto_pad;
-    InferenceEngine::Precision precision;
 
     std::string errorPrefix;
+
+    struct ExtractImagePatchesExecutor {
+        ExtractImagePatchesExecutor() {}
+        virtual void exec(void* src, void* dst, const VectorDims& istrides, const VectorDims& ostrides) = 0;
+        virtual const jit_extract_image_patches_params& getJpp() const = 0;
```

Comment:
as I can see `getJpp` used only inside `executor`
looks like no need for this method

Reply:
This method has been removed, thank you